source/extensions/filters/http/cache/http_cache.h

```diff
@@ -69,6 +71,10 @@ class RawByteRange {
   const uint64_t last_byte_pos_;
 };
 
+// Parses the ranges from the request headers into a vector<RawByteRange>.
+std::vector<RawByteRange> getRequestRanges(const Http::RequestHeaderMap& request_headers,
```

Comment:
optional: There aren't any response range headers, and this takes a RequestHeaderMap, so `Request` is implicit. This could be just `getRanges`.

Reply:
I changed it to parseRequests, since I created a class to enable the logger, wasn't a problem.